src/main/js/Widgets/AJAXDialog.js

```diff
@@ -197,6 +204,17 @@ class AJAXDialog {
     return this;
   }
 
+  /**
+   * Sets whether this dialog is dismissible by clicking off it or hitting Escape
```

Comment:
Re: closing when we lose focus (clicking off it) - I didn't realize we were also making this change. 

I'm ok with, and it is probably ok for now to re-use the same config for both options, but I could imagine we may want to separate out those things in the future.